dracut/30ignition/ignition-setup.sh

```diff
@@ -26,3 +28,7 @@ bootmnt=/mnt/boot_partition
 mkdir -p $bootmnt
 mount /dev/disk/by-label/boot $bootmnt
 copy_file_if_exists "${bootmnt}/ignition/config.ign" "${destination}/user.ign"
+
+if $(cmdline_bool 'coreos.no_persist_ip' 1); then
+    rm -rf /etc/sysconfig/network-scripts/*
```

Comment:
Just a thought, wouldn't this be under /sysroot ? Or does dracut actually persist to that location

Reply:
It would be under `sysroot/`; this PR is woefully out of date at the moment while I do fast iterations